message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1890,5 +1890,13 @@
       <field type="uint16_t[4]" name="rpm" units="rpm">RPM (eRPM).</field>
       <field type="uint16_t[4]" name="count">count of telemetry packets received (wraps at 65535).</field>
     </message>
+    <message id="11045" name="ARDUSOAR_EKF_ESTIMATE">
```

Comment:
```suggestion
    <message id="11045" name="ARDUSOAR_THERMAL_ESTIMATE">
```
Like so?

Reply:
Yes.  But perhaps we don't need "ARDUSOAR" in the title?

Also, have you evaluated `FW_SOARING_DATA` for its fitness?